addon/components/power-select.hbs

```diff
@@ -176,7 +177,6 @@
             id={{listboxId}}
             @ariaLabel={{@ariaLabel}}
             @ariaLabelledBy={{@ariaLabelledBy}}
-            @ariaSelected={{ember-power-select-is-selected opt @selected}}
```

Comment:
we didn't have access to the opt value here, so I had to do the calculation one level down in the options component